bin/coveralls.js

```diff
@@ -1,22 +1,23 @@
 #!/usr/bin/env node
-var handleInput = require('../lib/handleInput');
-var logger = require('../lib/logger');
```

Comment:
This seems to be unused. If not let me know and I'll change it to just `require('../lib/logger')`